Gemfile.lock

```diff
@@ -212,5 +212,8 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   web-console (>= 3.3.0)
 
+RUBY VERSION
+   ruby 2.5.3p105
+
```

Comment:
We're using the version 2.5.3 up on heroku, I'm not sure if this affects the local repos, but I think it's best to be operating on the same version of ruby anyways